src/packages.builds

```diff
@@ -10,7 +16,7 @@
     <Project Include="Native\pkg\**\*.builds">
       <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true</AdditionalProperties>
     </Project>
-    <Project Include="*\pkg\*.builds">
+    <Project Include="*\pkg\*.builds" Condition="'$(FilterToOSGroup)' == 'Windows_NT'">
```

Comment:
I think this should be enabled by default, and provide an opt-out property (`SkipPackages` or something). It it feels weird to special-case Windows here just because we're currently using Windows for our packages.